examples/conflicts/server.js

```diff
@@ -54,8 +56,9 @@ const resolvers = {
       greeting = conflict.nextState(args)
       return greeting
     },
-    changeGreetingClient: async (obj, args, { conflict }, info) => {
-      if (conflict.hasConflict(greeting, args, info)) {
+    changeGreetingClient: async (obj, args, context, info) => {
+      const { conflict } = context
+      if (conflict.hasConflict(greeting, args, obj, args, context, info)) {
```

Comment:
IMHO it will be cool to have separate function here for audit that can be used to pass that arguments. 
Alternatively we can mount all of those into context before and just pass only context. 
Alternatively we could aggregate them into single object - audit log.

Audit log is an additional feature here but it obfuscates conflict API so we may have couple quick wins.
Note: I'm not saying that we should do that now, but let's have discussion on this while we have PR opened.

Reply:
can you write some pseudo code? I didn't understand